### PR TITLE
feat(website): builder init

### PR DESCRIPTION
**Pull request by Itrulia**

### CODE REVIEW

File: libs/navigation/website/src/lib/navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const NavigationWrapper = styled(Box)`
+  flex-grow: 1;
+`
+
+const Title = styled.div`
+  flex-grow: 1;
+`
+
+export function Navigation({data, loading, error}: BuilderNavigationProps) {
+  const {Button} = useWebsiteBuilder()
```
Comment (Itrulia) [inline libs/navigation/website/src/lib/navigation.tsx:17]: An example how to get a component from the Website builder

File: libs/ui/jest.config.ts [hunk 2]
```diff
@@ -6,6 +6,6 @@ export default {
     '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
     '^.+\\.[tj]sx?$': ['babel-jest', {presets: ['@nrwl/react/babel']}]
   },
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
```
Comment (Itrulia) [inline libs/ui/jest.config.ts:9]: For storyshots

File: libs/ui/src/storyshots.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import {puppeteerTest} from '@storybook/addon-storyshots-puppeteer'
+
+initStoryShots({
+  configPath: resolve(__dirname, '../.storybook'),
+  integrityOptions: {cwd: join(__dirname, '../..')},
+  framework: 'react',
+  renderer: render
+})
+
+initStoryShots({
```
Comment (Itrulia) [inline libs/ui/src/storyshots.spec.ts:13]: Would automatically create design snapshots

File: libs/ui/src/storyshots.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import initStoryShots from '@storybook/addon-storyshots'
+import {join, resolve} from 'path'
+import {render} from '@testing-library/react'
+import {puppeteerTest} from '@storybook/addon-storyshots-puppeteer'
+
+initStoryShots({
```
Comment (Itrulia) [inline libs/ui/src/storyshots.spec.ts:6]: Would automatically create snapshots

File: libs/ui/src/storyshots.spec.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  suite: 'Puppeteer storyshots',
+  configPath: resolve(__dirname, '../.storybook'),
+  integrityOptions: {cwd: join(__dirname, '../..')},
+  framework: 'react',
+  test: puppeteerTest(),
+  renderer: render
+})
+
+it('currently storyshots does not work', () => {
```
Comment (Itrulia) [inline libs/ui/src/storyshots.spec.ts:22]: Stories can not be found currently

File: libs/website/src/lib/website.provider.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import {ThemeProvider, CssBaseline} from '@mui/material'
```
Comment (Itrulia) [inline libs/website/src/lib/website.provider.tsx:1]: Default provider split into a new library to avoid circular dependencies.

**Review by penk85 — APPROVED** [review]

**Pull request merged**